### ISSUE TITLE
Why was the poet.code project canceled?

### ISSUE THREAD

**Issue by jasp402** [krasimir/demoit#14]
I tried it when it first came out and created many stories and then it stopped working. I wanted to see this project emerge again and include AI to improve the concepts. I would like to know in the future it will come out again, if they can release all the Backend code to start a similar project from there or what the future projections are for this incredible project.

**Comment by krasimir** [comment]
Yeah ... I had some problems transitioning the domain from vercel to GCP and I left it as a task in my todo app. Some day I'll bring it back to life and will write here :)

**Comment by jasp402** [comment]
> Sí... tuve algunos problemas al realizar la transición del dominio de vercel a GCP y lo dejé como una tarea en mi aplicación de tareas pendientes. Algún día le devolveré la vida y escribiré aquí :)

Personally, I love this project. If I can help you in any way to revive it. Whatever you need, tell me and I will be giving you all the support I can.
